kube-aws: fix missing cluster-name in status cmd

cgag opened the pull request:
Info() never actually set the info.Name field to anything.

fixes #352

aaronlevy commented on multi-node/aws/cmd/kube-aws/command_validate.go:
```diff
@@ -25,7 +25,12 @@ var (
 
 func init() {
 	cmdRoot.AddCommand(cmdValidate)
-	cmdValidate.Flags().BoolVar(&validateOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
```
nit: I prefer oneliners for flags (more readable if there are multiple flags, so in this case not huge deal)

aaronlevy commented on multi-node/aws/pkg/cluster/cluster.go:
```diff
@@ -144,36 +144,19 @@ func (c *Cluster) Update(stackBody string) (string, error) {
+		},
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get public IP of controller instance")
```
Probably helpful to include the actual error text in the returned error

aaronlevy commented:
Couple minor comments.

However, I believe this might conflict with some changes proposed in https://github.com/coreos/coreos-kubernetes/pull/346 -- may need to coordinate with @colhom on timing.

cgag commented:
Rebased on Colin's #346 changes and added the aws error message to the error output. 

Some minor changes will need to be made after #362 is merged.

colhom commented:
LGTM for me.

colhom commented:
@cgag are we blocking for some reason on #362 before merging this one?

cgag commented:
I don't think so.  I think I made that comment when I had moved some stuff around, but that's been undone.
